package.json

```
@@ -70,4 +72,4 @@
     "@opam/opium": "github:EduardoRFS/opium:opium.opam#9fdbfbd0eedf238e1be0663ab95977be27d9a6e1",
     "@opam/rock": "github:EduardoRFS/opium:rock.opam#9fdbfbd0eedf238e1be0663ab95977be27d9a6e1"
   }
-}
+}
```

Can you fix this missing new line?
```suggestion
}

```